### PR TITLE
Pe-306

### PR DESCRIPTION
**Pull request by amalbas**
When the course has and url for payment, in the course about page you'll see the "buy now" button that link to the new url payment.

image

### CODE REVIEW

**Comment by amalbas** [comment]
@Squirrel18 i have to add the course about template from edx-platform, on second commit you can see my changes.

File: edx-platform/pearson-theme/lms/templates/courseware/course_about.html [hunk 1]
```diff
@@ -0,0 +1,528 @@
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from courseware.courses import get_course_about_section
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from edxmako.shortcuts import marketing_link
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangolib.markup import HTML
+from openedx.core.lib.courses import course_image_url
```
Comment (Squirrel18) [inline edx-platform/pearson-theme/lms/templates/courseware/course_about.html]: openedx are local application imports, so they need to be placed before six imports with a empty line between them. @amalbas  [PEP8 imports](https://www.python.org/dev/peps/pep-0008/#imports)

Comment (Squirrel18) [reply edx-platform/pearson-theme/lms/templates/courseware/course_about.html]: Sorry, openedx it was need to be placed after six imports. @amalbas

**Comment by Squirrel18** [comment]
Squash first, please. @amalbas

**Pull request merged**